README.md

```diff
@@ -87,6 +87,12 @@ On Linux and macOS, you can use this install script, which will automatically in
 curl -sSfL https://www.shuttle.rs/install | bash
 ```
 
+On Windows, you can use this install script to do the same:
+
+```powershell
+iwr "https://www.shuttle.rs/install-win" | iex
```

Comment:
nit: the quotes are not needed

Reply:
I don't mind removing them, but I had added them because I think the highlighting looks much better with:

<img width="302" alt="image" src="https://github.com/shuttle-hq/shuttle/assets/21363892/39f6d925-38b8-4d81-919f-b4aa312c9a93">

Vs without quotes:

<img width="287" alt="image" src="https://github.com/shuttle-hq/shuttle/assets/21363892/add09f98-63ac-4359-8faf-ffadac8220b8">